aws/ecs-service/main.tf

```diff
@@ -134,6 +134,9 @@ resource "aws_cloudwatch_metric_alarm" "service_count" {
   statistic           = "Average"
   treat_missing_data  = "ignore"
 
+  alarm_actions = [var.slack_topic_arn]
```

Comment:
Should this be some reference to an action variable or are we happy to always reference the slack topic?